### PR TITLE
Beta6 changelog

### PR DESCRIPTION
**Pull request by lboeman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,56 @@ in this file.
...
+- Temperature coefficients are now recorded in units of %/C.
+
+### Fixed
+
+- Report metric plots now display months and weekdays in the correct order.
+
+- Fixed incorrect timezones of DOE RTC sites.
+
+- Fixed multiple instances where the dashboard did not handle error responses
+  from the api as expected, causing crashes.
```
Comment (wholmgren) [inline CHANGELOG.md:58]:
```suggestion
  from the API as expected, causing crashes.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,56 @@ in this file.
...
+  Solar Forecast Arbiter's reference observation dataset.
+
+- Added support for derieved quality flags 'DAYTIME', 'DAYTIME STALE
+  VALUES' and 'DAYTIME INTERPOLATED VALUES'.
+
+- AC and DC power limits were added to the validation package. Values are
+  marked with the `LIMITS EXCEEDED` flag.
+
+- Automatically generated persistence forecasts were added to the Solar
+  Forecast Arbiter reference dataset.
```
Comment (wholmgren) [inline CHANGELOG.md:25]: move above quality flag items

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,56 @@ in this file.
 Detailed changes to the Solar Forecast Arbiter Core python library can be found
 in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.
 
+## [1.0beta6] - 2020-06-02
+
+### Added
+
+- Reports may now be downloaded in PDF format.
```
Comment (alorenzo175) [inline CHANGELOG.md:13]:
```suggestion
- Reports may now be downloaded in PDF format, although reports that currently exist must first be recomputed.
```

Comment (alorenzo175) [reply CHANGELOG.md:13]: otherwise 500 that we should handle but don't yet

**Pull request merged**